io.github.stella_emu.Stella.yaml

```
@@ -0,0 +1,32 @@
+app-id: io.github.stella_emu.Stella
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+
+rename-desktop-file: stella.desktop
+rename-icon: stella
+
+cleanup:
+  - /lib
+  - /share/doc
+
+finish-args:
+  - --filesystem=home:rw
+  - --socket=x11
```

```suggestion
  - --socket=fallback-x11
```

If it really works under Wayland, X11 should be only a fallback.